lib/jellyfish_web/server_socket.ex

```diff
@@ -83,13 +84,7 @@ defmodule JellyfishWeb.ServerSocket do
 
   def handle_in({encoded_message, [opcode: :binary]}, state) do
```

Comment:
```suggestion
  @impl true
  def handle_in({encoded_message, [opcode: :binary]}, state) do
```

Reply:
Nitpick; might as well annotate the `handle_in` callback from line 102.

Reply:
tbh, I prefer to make the annotations only once - above the first callback clause, but if everyone prefers adding annotations before each callback I will change this

Reply:
I think it'd be clearer this way, GPT4 seems to agree, but ultimately it's your decision